src/mount_efs/__init__.py

```diff
@@ -280,24 +279,8 @@
     MACOS_BIG_SUR_RELEASE,
     ORACLE_RELEASE_NAME,
     ALMALINUX_8_RELEASE_NAME,
+    ROCKY8_RELEASE_NAME
 ]
-=======
```

Comment:
I don't think this is a clean rebase right?

Reply:
It is not. I have completely messed up the history while squashing.